bazel/absl.patch

```diff
@@ -0,0 +1,12 @@
+diff -uw -r a/absl/base/options.h b/absl/base/options.h
+--- a/absl/base/options.h	2021-04-02 23:00:31.996584600 -0400
++++ b/absl/base/options.h	2021-04-02 23:01:07.506208600 -0400
+@@ -154,7 +154,7 @@
+ // absl::string_view is a typedef of std::string_view, use the feature macro
+ // ABSL_USES_STD_STRING_VIEW.
+
+-#define ABSL_OPTION_USE_STD_STRING_VIEW 2
```

Comment:
is this also an issue with `clang-cl` ? (I'd assume that clang might come with its own std:: library, though it is probably more likely that they use the Windows std:: headers for ABI compatibility).